example_notebooks/16-tls_tree_dynamic.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>Maybe we can write some times here (measured on a local computer)? They could be useful to understand the runtime difference between static and dynamic VLS. Otherwise just an "approximately x times longer" sentence could be enough as the execution times are strongly case dependent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>